config/main.py

```diff
@@ -48,7 +48,7 @@
 CFG_LOOPBACK_ID_MAX_VAL = 999
 CFG_LOOPBACK_NO="<0-999>"
 
-MAXIMUM_WARMRESTART_TIMER_VALUE = 9999
+DISABLE_WARMRESTART_TIMER_VALUE = 9999
```

Comment:
I believe it is possible to use the constant from swsscommon,

Reply:
we might be able to do it , at least we need to add  libswsscommon , python-swsscommon deb pkg and all the dependency (libnl-xxxx, hiredis, ...) into host environment.   Today only a few dockers (swss/vs/lldp/platform monitor) using python-swsscommon. There may be some conflicts if we have those hiredis in host , it may overwrite some command mounted to swss docker. I suggest keeping the current behaviors